src/app/pages/LandingPage/LandingPage.module.css

```diff
@@ -0,0 +1,54 @@
+@import '../../../../node_modules/augmented-ui/augmented-ui';
+
+.landingpage_body {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  background-image: url(/Users/patrickmoller/hh-web-21-7/capstone-project/src/images/background-app-1.png);
```

Comment:
This looks like the path on your Mac. It should be in a folder inside your app, I think.

Reply:
actually it is inside